deploy/examples/cephblockpoolradosnamespace-test.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: ceph.rook.io/v1
```

Comment:
The only "test.yaml" files needed are basically for pools to use replica 1, or when some other component needs to define a single daemon. Since there is nothing related to replica 1 in this file, let's revert this file.

Reply:
done